### PR TITLE
[dexter] Correctly identify stop-reason while driving VisualStudio

### PR DESCRIPTION
**Pull request by OCHyams**
Prior to this patch VisualStudio._get_step_info incorrectly identifies the reason the debugger has stopped. e.g., stepping through a program would be reported as a StopReason.Breakpoint rather than StopReason.Step.

Fix. No test added as there are no VisualStudio tests (tested locally).

### CODE REVIEW

File: cross-project-tests/debuginfo-tests/dexter/dex/debugger/visualstudio/VisualStudio.py [hunk 1]
```diff
@@ -307,6 +307,30 @@ def set_current_stack_frame(self, idx: int = 0):
                 )
             )
 
+    def _translate_stop_reason(self, reason):
+        # https://learn.microsoft.com/en-us/dotnet/api/envdte.dbgeventreason?view=visualstudiosdk-2022
+        if reason == 1:  # dbgEventReasonNone
```
Comment (CarlosAlbertoEnciso) [inline cross-project-tests/debuginfo-tests/dexter/dex/debugger/visualstudio/VisualStudio.py:312]: Can those magic values (1, 9, 8, ...) be replace for some named constants?

Comment (CarlosAlbertoEnciso) [reply cross-project-tests/debuginfo-tests/dexter/dex/debugger/visualstudio/VisualStudio.py:312]: May be something like:
```
dbgEventReasonNone = 1
dbgEventReasonGo = 2
...
if reason == dbgEventReasonNone: 
...
```
Basically is the same code, but I think it is easy to read.

Comment (SLTozer) [reply cross-project-tests/debuginfo-tests/dexter/dex/debugger/visualstudio/VisualStudio.py:312]: I don't have strong opinions, but in general even if it's only going to be used once I think it's probably better to have something like `class VSDebugEventReason(IntEnum):` with the correct values added to it.

**Review by SLTozer — APPROVED** [review]
Changes LGTM

**Review by CarlosAlbertoEnciso — APPROVED** [review]

**Pull request merged**